src/test/java/com/wikia/webdriver/testcases/communityheadertests/CommunityHeaderTests.java

```diff
@@ -51,30 +50,30 @@ public void testLoggedInWikiButtons() {
     CommunityHeader communityHeader = new CommunityHeader();
 
     communityHeader.clickWikiActivity();
-    Assert.assertTrue(driver.getCurrentUrl().contains("Special:WikiActivity"));
+    Assertion.assertTrue(driver.getCurrentUrl().contains("Special:WikiActivity"));
 
     CreateArticleModalComponentObject addNewPageModal = communityHeader.clickAddNewPage();
-    Assert.assertTrue(addNewPageModal.isCreateNewArticleModalVisible());
+    Assertion.assertTrue(addNewPageModal.isCreateNewArticleModalVisible());
     addNewPageModal.close();
 
     communityHeader.openMoreToolsDropdown()
         .clickMoreAddImageLink();
-    Assert.assertTrue(driver.getCurrentUrl().contains("Special:Upload"));
+    Assertion.assertTrue(driver.getCurrentUrl().contains("Special:Upload"));
 
     AddMediaModalComponentObject addVideoModal = communityHeader.openMoreToolsDropdown()
         .clickMoreAddVideoLink();
-    Assert.assertTrue(driver.getCurrentUrl().contains("Special:Videos"));
-    Assert.assertTrue(addVideoModal.isVideoModalVisible());
+    Assertion.assertTrue(driver.getCurrentUrl().contains("Special:Videos"));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "Special:Videos" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)